test/transformationsTest.js

```diff
@@ -0,0 +1,291 @@
+'use strict'
+
+const test = require('tape')
+const transforms = require('../src/transform/handler')
+
+test.test('Transformations', t => {
+  t.test('dateTimeToDate()', t => {
+    t.test('should convert iso datetime string to date', t => {
+      const inputDate = '1988-03-24T00:00:00.000+0800'
+      const expected = '1988-03-24'
+
+      const result = transforms['dateTimeToDate'](inputDate)
+
+      t.equal(result, expected)
+      t.end()
+    })
+  })
+
+  t.test('getGenderFromCode()', t => {
+    t.test('should convert single letter code to word', t => {
+      const inputCode = 'F'
+      const expectedOutput = 'female'
+
+      const functionParameters = {
+        F: 'female',
+        M: 'male',
+        O: 'other'
+      }
+
+      const result = transforms['mapCodes'](
+        inputCode,
+        null,
+        null,
+        null,
+        null,
+        functionParameters
+      )
+
+      t.equal(result, expectedOutput)
+      t.end()
+    })
+
+    t.test('should convert unhandled code to null', t => {
+      const inputCode = 'F'
+      const expectedOutput = 'female'
+
+      const functionParameters = {
+        F: 'female',
+        M: 'male',
+        O: 'other'
+      }
+
+      const result = transforms['mapCodes'](
+        inputCode,
+        null,
+        null,
+        null,
+        null,
+        functionParameters
+      )
+
+      t.equal(result, expectedOutput)
+      t.end()
+    })
+
+    t.test(
+      'should convert unhandled codes to default value if specified',
+      t => {
+        const inputCode = 7
+        const expectedOutput = 'unknown'
+        const functionParameters = {
+          F: 'female',
+          M: 'male',
+          O: 'other',
+          default: 'unknown'
+        }
+
+        const result = transforms['mapCodes'](
+          inputCode,
+          null,
+          null,
+          null,
+          null,
+          functionParameters
+        )
+
+        t.equal(result, expectedOutput)
+        t.end()
+      }
+    )
+
+    t.test(
+      'should convert unhandled codes to null if no default specified',
+      t => {
+        const inputCode = 7
+        const expectedOutput = undefined
```

Comment:
Should this be `undefined` or `null`?

Reply:
Undefined :| 
I'm not sure if that will be an issue or not for object-mapper... I would assume not?

Reply:
Im not to sure but dont think so, just need to update the test title to state `undefined` insted of `null` :)